### PR TITLE
intial commit

### PR DESCRIPTION
**Pull request by swethaogeti**
added homepage login/signup

### CODE REVIEW

**Comment by swethaogeti** [comment]
for preview check: https://agitated-banach-1de3c2.netlify.app/

**Review by Debarshi95 — APPROVED** [review]
Looks good. Can merge.

File: components/form.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    color: var(--primary-dark);
+    width: 40rem;
+    height: 40rem;
+    margin: auto;
+    margin-top: 10rem;
+    box-shadow: rgba(0, 0, 0, 0.1) 0px 4px 12px;
```
Comment (theVivek-kumar) [inline components/form.css:11]: use var( )

File: components/productlisting.css [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+.productlisting .main .card-container h4 {
+  font-size: 2.3rem;
+}
+
+.productlisting .main .card-container p {
+  font-size: 1.5rem;
+  text-align: center;
+  font-weight: 400;
+  margin-top: 1rem;
+}
```
Comment (theVivek-kumar) [inline components/productlisting.css:70]: this style convention is good

**Review by theVivek-kumar — APPROVED** [review]

File: components/form.css (hunk 1 above)
Comment (swethaogeti) [reply components/form.css:11]: done

**Pull request closed without merging**